packages/dd-trace/test/debugger/devtools_client/condition-test-cases.js

```diff
@@ -845,7 +848,7 @@ function createClassWithOverloadedMethodInPrototypeChain (Builtin, propName) {
  *  collectionOperations: TestCase[],
  *  membershipAndMatching: TestCase[],
  *  typeAndDefinitionChecks: TestCase[]
- * }} */
+  }} */
```

Comment:
I think this looks nicer

```suggestion
 * }} 
 */
```